### PR TITLE
build: move libcc patches to electron repo

### PR DESCRIPTION
**Pull request by nornagon**
In the GN build, libchromiumcontent is no longer a distinct library, but
merely a container for a set of scripts and patches. Maintaining those
patches in a separate repository is tedious and error-prone, so merge
them into the main repo.

Once this is merged and GN is the default way to build Electron, the
libchromiumcontent repository can be archived.

Depends on #14097.

- [x] PR description included and stakeholders cc'd
- [x] PR title follows semantic [commit guidelines](https://github.com/electron/electron/blob/master/docs/development/pull-requests.md#commit-message-guidelines)

Notes: no-notes

### CODE REVIEW

**Comment by alexeykuzmin** [comment]
I'm going to touch lots of libcc files in https://github.com/electron/libchromiumcontent/pull/647 =/

**Comment by nornagon** [comment]
Rebased on top of M67 merge.

**Comment by alexeykuzmin** [comment]
@nornagon Can you please move [script/README.md](https://github.com/electron/libchromiumcontent/blob/master/script/README.md) too?

**Comment by alexeykuzmin** [comment]
@jkleinsc Do we still use `sccache` binaries from the libcc repo?

File: DEPS [hunk 1]
```diff
@@ -23,22 +27,22 @@ vars = {
...
+  'src/electron/vendor/pyyaml':
+    (Var("yaml_git")) + '/pyyaml.git@' + (Var("pyyaml_version")),
 }
 
 hooks = [
   {
     'action': [
       'python',
-      'src/libchromiumcontent/script/apply-patches',
+      'src/electron/script/apply-patches'
```
[the remark this answers is not in the document]
Comment (nornagon) [reply DEPS]: oops! thanks!

**Pull request merged**